src/main/java/com/ust/sourcecourse/configuration/service/DBDataSourceService.java

```diff
@@ -169,7 +184,10 @@ public List<String> addTagSourceTable(Long uid, List<String> tags, String descri
 
 	public List<String> addTagSourceColumn(Long uid, List<String> tags, String description) {
 		SourceColumn sourceColumn = sourceColumnRepository.findById(uid)
-				.orElseThrow(() -> new ResourceNotFoundException("sourceColumn", "id", uid));
+				.orElseThrow(() -> new CustomException.ResourceNotFoundException("sourceColumn", "id", uid));
+		if (description == null || tags == null) {
+			throw new IllegalArgumentException("Description and tags cannot be null");
```

Comment:
Either one can be null. Description or tags is not mandatory.

Reply:
resolved